db/migrate/20210110112042_backfill_feedback_column.rb

```diff
@@ -0,0 +1,5 @@
+class BackfillFeedbackColumn < ActiveRecord::Migration[6.0]
+  def change
+    ApplicationFeedback.all.each { |feedback| feedback.update!(feedback: feedback.other_feedback) }
```

Comment:
This is really neat 👍🏽

Reply:
Could we use `update_column` so we don't touch `updated_at`? https://davidverhasselt.com/set-attributes-in-activerecord/